api/orm/db_init.py

```diff
@@ -0,0 +1,21 @@
+""" Setup database to perform CRUD transactions
+"""
```

Comment:
What did you model the file structure after? I'm looking at https://fastapi.tiangolo.com/tutorial/sql-databases/ and see they're using "database.py", "models.py" etc. any reason for deviating from that?

Reply:
Would re-structure as per EPR #19